smoke_tests/ecs_fargate/variables.tf

```
@@ -18,4 +18,10 @@ variable "dd_site" {
   description = "Datadog Site"
   type        = string
   default     = "datadoghq.com"
+}
+
+variable "test_prefix" {
+  description = "The ECS task family name prefix"
+  type        = string
+  default     = "terraform-test"
 }
```

small linter issue that was present before 

```suggestion
}

```